frigate/record.py

```diff
@@ -97,8 +97,21 @@ def move_files(self):
                 os.makedirs(directory)
 
             file_name = f"{start_time.strftime('%M.%S.mp4')}"
+            file_path = os.path.join(directory, file_name)
 
-            os.rename(os.path.join(RECORD_DIR, f), os.path.join(directory, file_name))
+            os.rename(os.path.join(RECORD_DIR, f), file_path)
+
+            rand_id = "".join(
+                random.choices(string.ascii_lowercase + string.digits, k=6)
+            )
+            Recordings.create(
+                id=f"{start_time.timestamp()}-{rand_id}",
```

Comment:
Not currently, but eventually we can refactor the maintainer to pull from this table instead of looking at disk, then do a delete by ID instead of path.